### PR TITLE
Adding a new compset for MAML

### PR DESCRIPTION
**Pull request by lee1046**
2 compsets are added 1) to run MAML (compset name: SP1V1-MAML), 2) to run MMF tests with MAML (compset name: SP1V1-MAML-TEST). Standalone test is not done yet, will do it tomorrow once cori is back.

### CODE REVIEW

File: components/cam/cime_config/config_component.xml [hunk 1]
```diff
@@ -105,6 +105,9 @@
       <value compset="_CAM5%SP2V1"           >-SPCAM_microp_scheme m2005 -chem linoz_mam4_resus_mom_soag -rain_evap_to_coarse_aero </value>
       <value compset="_CAM5%SP[12]V1_"       >-crm_nx 64 -crm_ny 1 -crm_nx_rad 16 -crm_ny_rad 1 </value>
       <value compset="_CAM5%SP[12]V1-TEST_"  >-crm_nx  8 -crm_ny 1 -crm_nx_rad  1 -crm_ny_rad 1 </value>
+      <value compset="_CAM5%SP1V1-MAML"      >-use_MAML -crm_ny 1 -crm_ny_rad 1 </value>
+      <value compset="_CAM5%SP1V1-MAML_"     >-crm_nx 64 -crm_nx_rad 64 </value>
```
Comment (whannah1) [inline components/cam/cime_config/config_component.xml]: Should we just combine these lines?

Comment (lee1046) [reply components/cam/cime_config/config_component.xml]: I had it combined originally and only found out that the final CAM_CONFIG_OPT had both -crm_nx_rad 16 and -crm_nx_rad 64, so I had to break it up.

File: components/cam/cime_config/config_component.xml [hunk 2]
```diff
@@ -374,20 +377,25 @@
     <desc compset="CAM[45].*_BGC%B">CAM prognostic CO2 cycle turned on.</desc>
     <!-- Super-Parameterization -->
     <desc compset="_CAM5%SP1V1_"    >Super-parameterized E3SM with 64x1km CRM, RRTMG radiation, 1-moment microphysics and prescribed aerosol</desc>
+    <desc compset="_CAM5%SP1V1-MAML">Super-parameterized E3SM with 64x1km CRM, RRTMG radiation, 1-moment microphysics, prescribed aerosol and MAML </desc>
```
Comment (whannah1) [inline components/cam/cime_config/config_component.xml]: Let's include a note that the time step defaults to 5 min (since this means it's somewhat specific to ne30 and needs to be reduced further for ne120) and that the reduced rad is disabled.

File: components/cam/cime_config/config_compsets.xml [hunk 3]
```diff
@@ -371,18 +371,28 @@
     <alias>FSP1V1</alias>
     <lname>2000_CAM5%SP1V1_CLM45%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
   </compset>
+  
+  <compset>
+    <alias>FSP1V1-MAML</alias>
+    <lname>2000_CAM5%SP1V1-MAML_CLM45%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV</lname>
```
Comment (whannah1) [inline components/cam/cime_config/config_compsets.xml]: A note about SPBC - I think the "BC" part is why we've had some confusion about the "bc_dep_to_snow_updates" flag. I think we can omit this flag if we take off the "BC" for 1-mom runs since we are using prescribed aerosols. We can do this in another PR though since I'm sure it will be non-BFB.
